### PR TITLE
Add mailgun integration api docs

### PR DESCRIPTION
**Pull request by hebime**

### CODE REVIEW

File: slate/source/includes/rest/managed/_integrations.md [hunk 1]
```diff
@@ -286,21 +286,23 @@ curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6/integrations \
...
 
-| **Arguments**             |   |
-|---------------------------|---|
-| **type**<br/><span class='req'>required</span> | The integration type: _frontendEmail_. |
-| **fromAddress**<br/><span class='opt'>optional</span> | Email will display as coming from this address. |
+| **Arguments**                                             |                                                                                                      |
+|-----------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| **type**<br/><span class='req'>required</span>            | The integration type: _mailgun_.                                                                     |
+| **apiKey**<br/><span class='req'>required</span>          | The public API key of your Mailgun account.                                                          |
+| **domain**<br/><span class='req'>required</span>          | One of the domains you have configured in Mailgun.                                                   |
```
Comment (jpjoyal) [inline slate/source/includes/rest/managed/_integrations.md]: "One of the domains you have configured in Mailgun. " -> The domain to use to relay email. This domain must be configured and verified in your Mailgun account.

File: slate/source/includes/rest/managed/_integrations.md [hunk 2]
```diff
@@ -286,21 +286,23 @@ curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6/integrations \
...
+| **incomingAddress**<br/><span class='req'>required</span> | Smooch will receive all emails sent to this address. It will also be used as the *Reply-To* address. |
```
Comment (jpjoyal) [inline slate/source/includes/rest/managed/_integrations.md]: Maybe add a not about the fact that it must be an email within the specified domain ? Do we actually validate that ?

Comment (hebime) [inline slate/source/includes/rest/managed/_integrations.md]: Yea we check that. I can add that note.

File: slate/source/includes/rest/managed/_integrations.md (hunk 1 above)
Comment (hebime) [inline slate/source/includes/rest/managed/_integrations.md]: Perfect!

File: slate/source/includes/rest/managed/_integrations.md [hunk 3]
```diff
@@ -286,21 +286,25 @@ curl https://api.smooch.io/v1/apps/55c8d9758590aa1900b9b9f6/integrations \
...
-    "integration": {
-        "_id": "5735ddfd48011972d621dc0a",
-        "type": "frontendEmail",
-        "shortId": "tki1106",
-        "smoochAddress": "app.tki1106@mail.smooch.io"
-    }
+	"integration": {
+		"type": "mailgun",
+		"domain": "sandbox123.mailgun.org",
+		"fromAddress": "mytestemail@sandbox123.mailgun.org",
```
Comment (dannytranlx) [inline slate/source/includes/rest/managed/_integrations.md]: Still saying `fromAddress` here

**Comment by hebime** [comment]
Nice catch! @dannytranlx

File: slate/source/includes/rest/managed/_integrations.md (hunk 3 above)
Comment (hebime) [inline slate/source/includes/rest/managed/_integrations.md]: Nice catch!

**Pull request merged**